### PR TITLE
sshd: change default config

### PR DESCRIPTION
**Pull request by tjjh89017**
PermitRootLogin no
LoginGraceTime 60
AllwoGroups admin

Signed-off-by: Date Huang <date.huang@suse.com>

### CODE REVIEW

File: files/system/oem/09_sshd.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: "sshd"
```
Comment (bk201) [inline files/system/oem/09_sshd.yaml]: it would be nice to describe the purpose more, not just "sshd"

File: files/system/oem/09_sshd.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+stages:
+   initramfs:
+     - name: "clear origin config"
```
Comment (bk201) [inline files/system/oem/09_sshd.yaml]: nit: original.

Besides this, The feature works great, thanks.

**Comment by tjjh89017** [comment]
update name for descritbe

**Comment by bk201** [comment]
@tjjh89017 Just found the file continues to grow between yip runs, do we want to tune this?
```
### Harvester Config
#PermitRootLogin no
#LoginGraceTime 60
#AllowGroups admin
### Harvester Config
#PermitRootLogin no
#LoginGraceTime 60
#AllowGroups admin
### Harvester Config
PermitRootLogin no
LoginGraceTime 60
AllowGroups admin
```

to test:
```
yip -s initramfs <the yaml file>
```

**Comment by tjjh89017** [comment]
> @tjjh89017 Just found the file continues to grow between yip runs, do we want to tune this?
> 
> ```
> ### Harvester Config
> #PermitRootLogin no
> #LoginGraceTime 60
> #AllowGroups admin
> ### Harvester Config
> #PermitRootLogin no
> #LoginGraceTime 60
> #AllowGroups admin
> ### Harvester Config
> PermitRootLogin no
> LoginGraceTime 60
> AllowGroups admin
> ```
> 
> to test:
> 
> ```
> yip -s initramfs <the yaml file>
> ```

I will expect this script is only executed when the system is boot up.
I'll try to fix this.

**Comment by tjjh89017** [comment]
It should not continue growing between yip runs.

**Review by Vicente-Cheng — APPROVED** [review]
LGTM

**Review by bk201 — APPROVED** [review]
Thanks for addressing the suggestion.

**Pull request merged**